src/System.Data.DataSetExtensions/tests/Mono/DataRowComparerTest.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I believe for files originating from Mono, we need to not include this 2nd line of the header (but keep the 1st and 3rd).  @JeremyKuhne can confirm.